nucliadb/nucliadb/search/search/merge.py

```diff
@@ -62,22 +63,65 @@
     ResourceSearchResults,
     Sentence,
     Sentences,
+    SortField,
+    SortOptions,
+    SortOrder,
     TextPosition,
 )
 
+Score = Any
```

Comment:
Maybe it looks ugly, but you can be more specific about the type for score.
Something like:

```python
NodeScore = Tuple[int, int]
TimestampScore = datetime.datetime
TitleScore = str

Score = Union[NodeScore, TimestampScore, TitleScore]
```

Reply:
Sounds nicer, changed